internal/config/load.go

```diff
@@ -87,6 +111,34 @@ func initDefaultConfigFile() {
 	}
 }
 
+func findLocalConfig(startPath string, configFileName string) (string, bool) {
+	wd := startPath
+	checked := ""
+	found := hasConfigFile(wd, configFileName)
+	for !found && wd != checked {
+		checked = wd
+		wd = filepath.Dir(wd)
+		found = hasConfigFile(wd, configFileName)
+	}
+
+	if found {
+		return wd, true
+	} else {
+		return "", false
+	}
+}
+
+func hasConfigFile(configRootPath string, configFileName string) bool {
+	_, err := os.Stat(filepath.Join(configRootPath, configFileName))
+	if err == nil {
+		return true
+	}
+	if os.IsNotExist(err) {
```

Comment:
If the default behavior is to return false, I am not sure this check is necessary

Reply:
Fixed